src/pagerduty.js

```diff
@@ -275,11 +278,54 @@ function processSchedulesFromConfig(done) {
   });
 }
 
+function getEngineersOncall(scheduleId, escalationPolicyId, done) {
+  const extraOptions = {
+    qs: {
+      'include[]': 'users',
+      'escalation_policy_ids[]': escalationPolicyId,
+    },
+  };
+  getSchedule(scheduleId, extraOptions, (err, scheduleInfo) => {
+    if (err) { return done(err); }
+    const timeToCheck = moment(nconf.get('TIME_SINCE')) || moment();
```

Comment:
this is that time of handover?

Reply:
this is the time of execution of the task; I'm using the legacy function that returns oncalls for the whole configured interval (2 weeks), can be optimized in the next iteration